packages/beacon-node/src/chain/stateCache/persistentCheckpointsCache.ts

```diff
@@ -212,20 +212,7 @@ export class PersistentCheckpointStateCache implements CheckpointStateCache {
       }
       sszTimer?.();
       const timer = this.metrics?.stateReloadDuration.startTimer();
-      const newCachedState = loadCachedBeaconState(
-        seedState,
-        stateBytes,
-        {
-          shufflingGetter: (shufflingEpoch, decisionRootHex) => {
-            const shuffling = this.shufflingCache.getSync(shufflingEpoch, decisionRootHex);
-            if (shuffling == null) {
-              this.metrics?.stateReloadShufflingCacheMiss.inc();
-            }
-            return shuffling;
-          },
-        },
-        validatorsBytes
-      );
+      const newCachedState = loadCachedBeaconState(seedState, stateBytes, {}, validatorsBytes);
```

Comment:
why remove the `shufflingGetter` here? it's used to speed up `loadCachedBeaconState` api

Reply:
nvm, we have shuffling cache inside EpochCache of state-transition already

Reply:
Yep exactly.  I switched to using the ShufflingCache now.